src/command_modules/azure-cli-appservice/setup.py

```diff
@@ -37,7 +37,7 @@
     'urllib3[secure]>=1.18',
     'xmltodict',
     'pyOpenSSL',
-    'vsts-cd-manager',
+    'vsts-cd-manager<=0.118.0',
```

Comment:
You should probably lock this down to < 1.0 - you may need to patch the 0.118 version later for some reason.